### PR TITLE
Ci and docker improvement

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -125,32 +125,21 @@ jobs:
...
-        if: matrix.target == 'aarch64-unknown-linux-gnu'
         run: |
           sudo apt-get update
           sudo apt-get install gcc-aarch64-linux-gnu
-      - run: cargo build --target=${{ matrix.target }} --bins
+      - run: cargo build --target=aarch64-unknown-linux-gnu --bins
       - name: Upload binaries
         uses: actions/upload-artifact@v2
         with:
           name: maker-and-taker-binaries-${{ matrix.target }}
```
Comment (bonomat) [inline .github/workflows/ci.yml]: Fixed in a fixup commit.

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
-FROM debian:bullseye-slim
+FROM gcr.io/distroless/cc
```
Comment (thomaseizinger) [inline Dockerfile:1]: :heart_eyes: 

How small is the image after this?

**Review by thomaseizinger — APPROVED** [review]

**Comment by thomaseizinger** [comment]
I assume you tested this on a fork? :)

**Review by da-kami — APPROVED** [review]

**Comment by bonomat** [comment]
> I assume you tested this on a fork? :)

Yap, tested on my own fork : https://github.com/bonomat/hermes/actions/runs/1505945379

File: Dockerfile (hunk 2 above)
Comment (bonomat) [reply Dockerfile:1]: image

The one on the top is the new one :)

Comment (thomaseizinger) [reply Dockerfile:1]: Nice! Still bigger than I expected but that is pretty good :)

**Pull request merged**